src/Orders/OrderDetailsScreen.js

```diff
@@ -0,0 +1,301 @@
+/* eslint-disable no-plusplus */
```

Comment:
helen would probably say to remove this somehow

Reply:
ok i fixed it for the details screen (but left it for the OrderScreen because the solution is more messy than leaving it there)